search_that_hash/cracker/greppable_mod/runner.py

```diff
@@ -0,0 +1,25 @@
+from search_that_hash.cracker.sth_mod import sth
+from search_that_hash.cracker import cracking
+import json
+
+
+class main:

```

Comment:
This is  an extremely bad class name, it's not main at all. Try `GreppableClass` or something?